kyuubi-metrics/src/main/scala/org/apache/kyuubi/metrics/MetricsConf.scala

```diff
@@ -34,26 +34,39 @@ object MetricsConf {
       .createWithDefault(true)
 
   val METRICS_REPORTERS: ConfigEntry[Seq[String]] = buildConf("metrics.reporters")
-    .doc("A comma separated list for all metrics reporters<ul>" +
-      " <li>JSON - default reporter which outputs measurements to json file periodically</li>" +
-      " <li>CONSOLE - ConsoleReporter which outputs measurements to CONSOLE.</li>" +
-      " <li>SLF4J - Slf4jReporter which outputs measurements to system log.</li>" +
-      " <li>JMX - JmxReporter which listens for new metrics and exposes them as namespaced" +
-      " MBeans.</li> </ul>")
+    .doc("A comma separated list for all metrics reporters" +
+      "<ul>" +
+      " <li>CONSOLE - ConsoleReporter which outputs measurements to CONSOLE periodically.</li>" +
+      " <li>JMX - JmxReporter which listens for new metrics and exposes them as MBeans.</li> " +
+      " <li>JSON - JsonReporter which outputs measurements to json file periodically.</li>" +
+      " <li>SLF4J - Slf4jReporter which outputs measurements to system log periodically.</li>" +
+      "</ul>")
     .version("1.2.0")
     .stringConf
     .transform(_.toUpperCase())
     .toSequence
-    .createWithDefault(Seq(JSON.toString))
+    .createWithDefault(Seq(SLF4J.toString))
```

Comment:
this is a bit dangerous change. the server log might increase exponentially

Reply:
Got it, will revert it.